content/source/docs/enterprise/sentinel/examples.html.md

```diff
@@ -12,6 +12,7 @@ and are far from exhaustive.
 
 ### Amazon Web Services
 
+* [Enforce owner allow list on `aws_ami` data source](https://github.com/hashicorp/terraform-guides/blob/master/governance/aws/enforce-ami-owners.sentinel)
```

Comment:
This sounds a little weird but I tried to keep the list order 'super generic' => 'more specific', think this might make more sense at the end of the list?

Reply:
Cool, sounds good then!